reproject/interpolation/high_level.py

```diff
@@ -82,5 +92,23 @@ def reproject_interp(input_data, output_projection, shape_out=None, hdu_in=0,
     if isinstance(order, str):
         order = ORDER[order]
 
-    return _reproject_full(array_in, wcs_in, wcs_out, shape_out=shape_out, order=order,
-                           array_out=output_array, return_footprint=return_footprint)
+    # if either of these are not default, it means a blocked method must be used
+    if block_size is not None or parallel is not False:
+        # if parallel is set but block size isn't, we'll choose
+        # block size so each thread gets one block each
+        if parallel is not False and block_size is None:
+            block_size = [dim // os.cpu_count() for dim in shape_out]
```

Comment:
This code block doesn't match the comment above, right?  If we have 4 CPUs, say, and a 2D image, `block_size` will be 1/16th the image size, so each CPU will get 4 blocks.  This isn't necessarily a bad thing, but it's also not obviously the right default.

Reply:
Ah yes you're dead right, and its probably worth doing big long strips that match the fits memory layout anyway.